### PR TITLE
B2-183-case-study-nav

### PR DESCRIPTION
**Pull request by EricWaight**
**feat: Case Study Bottom Nav Component**
Closes [B2-183](https://b2io.atlassian.net/browse/B2-183)
- Added remaining content to Net Jets case study
- Added component for bottom navigation

Note: This feature can be viewed on the Net Jets case study page

**Update:**
- There is still a console warning I am aware of and will address in a separate PR
- Laura and I discussed some topics around the techlist component, we decided to keep it below the main image on the case study pages, this was the intended design for mobile so it didn't make sense for us to do a lot more work to put it in a different place just for desktop
- Content for all these case studies needs audited
- Additional case study logos could be added at a later time

**Update 2:**
- Discussed console warning "Warning: Prop `id` did not match." with Drew, this is a React Hydration issue that will only occur in development and not in production so no change is necessary at this time

### CODE REVIEW

File: src/components/CaseStudy/CaseStudyResultsPane.tsx [hunk 1]
```diff
@@ -33,18 +34,24 @@ export const CaseStudyResultsResultsPane: FC<CaseStudyResultsPaneProps> = ({
...
-          margin: 0 -${spacing.xxl};
-          padding: ${cssClamp([10.5, 'mobile'], [1.5, 'desktop'])} ${spacing.xxl};
+          padding: ${cssClamp([10.5, 'mobile'], [1.5, 'desktop'])}
+            ${spacing.xxl};
         }
 
         ${atMinXXL} {
           padding-left: ${spacing.marginXl};
           padding-right: ${spacing.marginXl};
+          margin-left: calc(50% - 50vw);
```
Comment (tborres) [inline src/components/CaseStudy/CaseStudyResultsPane.tsx:50]: ☝️ move this up for alpha order

**Review by tborres — COMMENTED** [review]
The changes look good, but I did notice an issue on smaller screens. On smaller screens (Pixel 5 in this case) the text and images are cut off.

image

**Review by tborres — APPROVED** [review]
Just a comment about the responsive breakpoints for the bottom navigation. Take it into consideration, but otherwise this is ready to merge up.

We can review the full accuracy and verbiage of the content at the feature-branch-level.

File: src/components/CaseStudy/CaseStudyBottomNav.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+}) => {
+  return (
+    <section
+      css={css`
+        display: flex;
+        gap: ${spacing.lg};
+        justify-content: center;
+        margin-top: ${spacing.xxl2};
+
+        ${atMaxLg} {
```
[the remark this answers is not in the document]
Comment (EricWaight) [reply src/components/CaseStudy/CaseStudyBottomNav.tsx]: I agree, I made a couple adjustments if you wanna take a look

[the remark this answers is not in the document]
Comment (tborres) [reply src/components/CaseStudy/CaseStudyBottomNav.tsx]: :shipit:

**Pull request merged**